### PR TITLE
spring boot included

### PR DESCRIPTION
**Pull request by d--**

### CODE REVIEW

File: doge-webapp/src/test/java/main/doge/DogeWebappApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class DogeWebappApplicationTests {
+
```
Comment (phasebash) [inline doge-webapp/src/test/java/main/doge/DogeWebappApplicationTests.java:16]: @Autowired the webapplicationcontext and assertNotNull?

File: doge-webapp/build.gradle [hunk 2]
```diff
@@ -1,7 +1,23 @@
 description = "doge webapp"
 
 apply plugin: 'war'
+apply plugin: 'org.springframework.boot'
+
+buildscript {
+    ext {
```
Comment (phasebash) [inline doge-webapp/build.gradle:7]: test comment

**Review by phasebash — CHANGES_REQUESTED** [review]
must reply back with `meatspin`

File: doge-webapp/build.gradle (hunk 2 above)
Comment (d--) [reply doge-webapp/build.gradle:7]: `meatspin`

File: doge-webapp/src/test/java/main/doge/DogeWebappApplicationTests.java (hunk 1 above)
Comment (d--) [reply doge-webapp/src/test/java/main/doge/DogeWebappApplicationTests.java:16]: Will do.

Comment (d--) [reply doge-webapp/src/test/java/main/doge/DogeWebappApplicationTests.java:16]: Fixed.

**Review by phasebash — APPROVED** [review]

**Pull request merged**